adrs/3610135900/README.md

```diff
@@ -0,0 +1,64 @@
+# Secret Value Syntax
+
+- **State:** Draft
+- **Created:** 2023-08-13
+- **Tags:** secret
+
+## Context
+
+The current ecosystem lacks a standardized format for secret values. This
+decentralization of secret format decisions leads to a situation where
+developers often define the format themselves. Such a disparate approach results
+in many developers either forgetting to set up scanning tools or being unable to
+utilize preconfigured scanning tools.
+
+The lack of a consistent secret format introduces vulnerabilities and potential
+security risks. The changes in GitHub Authentication Token Formats, among other
```

Comment:
> The lack of a consistent secret format introduces vulnerabilities and potential security risks.


I don't buy this line.  

🤔 If you perfectly protect your secrets, then they can be in any format you want, and the result would be the same, zero exposure.   However, once "the mistake" does happen and the vulnerability is introduced, the lack of a consistent format undermines security scans.


```suggestion
The lack of a consistent secret format hinders tools that would otherwise help guard against vulnerabilities and potential
security risks. The changes in GitHub Authentication Token Formats, among other
```